Exit normally for help/verison options

If -v or -h is used for the help/version information, it is a normal exit situation,
not an error condition. Sometimes these are used as a simple operation test of the
resulting binary so the exit code does matter.

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
Upstream-Status: Submitted [https://salsa.debian.org/minicom-team/minicom/-/merge_requests/14]
---
 src/minicom.c |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

Index: minicom-2.8/src/minicom.c
===================================================================
--- minicom-2.8.orig/src/minicom.c
+++ minicom-2.8/src/minicom.c
@@ -1257,14 +1257,14 @@ int main(int argc, char **argv)
                  "modify it under the terms of the GNU General Public License\n"
                  "as published by the Free Software Foundation; either version\n"
                  "2 of the License, or (at your option) any later version.\n\n"));
-          exit(1);
+          exit(0);
           break;
         case 's': /* setup mode */
           dosetup = 1;
           break;
         case 'h':
           helpthem();
-          exit(1);
+          exit(0);
           break;
         case 'p': /* Pseudo terminal to use. */
           if (strncmp(optarg, "/dev/", 5) == 0)
